src/TheFox/FlickrCli/Command/UploadCommand.php

```diff
@@ -76,11 +76,12 @@ protected function configure()
         $this->setName('upload');
         $this->setDescription('Upload files to Flickr.');
 
+        $csvDesc = 'Comma separated names. For example: --sets=set1,set2';
         $this->addOption('config', 'c', InputOption::VALUE_OPTIONAL, 'Path to config file. Default: config.yml');
         $this->addOption('log', 'l', InputOption::VALUE_OPTIONAL, 'Path to log directory. Default: log');
         $this->addOption('description', 'd', InputOption::VALUE_OPTIONAL, 'Description for all uploaded files.');
-        $this->addOption('tags', 't', InputOption::VALUE_OPTIONAL, 'Comma separated names. For example: --tags=tag1,tag2');
-        $this->addOption('sets', 's', InputOption::VALUE_OPTIONAL, 'Comma separated names. For example: --sets=set1,set2');
+        $this->addOption('tags', 't', InputOption::VALUE_OPTIONAL, $csvDesc);
```

Comment:
The string in `$csvDesc` is not valid for option `tags`. There is a separate example for tags and sets.